style.css

```diff
@@ -26,23 +46,38 @@ body {
 
 a {
     color: inherit;
-    text-shadow: 0 1px 2px white;
+    font-weight: 400;
+}
+
+b, strong {
+    /* Firefox uses "bolder" be default, which is... not bold enough */
+    font-weight: bold;
 }
 
 h1 {
     font-family: 'Nova Square', sans-serif;
     font-size: 3em;
     font-weight: normal;
     color: #fff3ed;
-    text-shadow: 0 1px 2px #2f2f2e;
     text-align: center;
     margin: 0;
     margin-bottom: 0.25rem;
     padding-top: 1.25rem;
 }
 
 h2 {
-    font-family: 'Cinzel', monospace;
+    font-family: 'Charter', 'XCharter', 'Bitstream Charter', serif;
```

Comment:
Charter isn't being imported.

Reply:
Charter is considered a web-safe font because it ships with Windows, macOS, and iOS. XCharter seems to be another common name for it when installed on Linux systems.

Source: https://iainbean.com/posts/2021/system-fonts-dont-have-to-be-ugly/#Charter

Reply:
Weird.  I suspected that was the intent so I checked my freshly-installed system and found no font by this name...

Reply:
I tend to agree with Deledrius; I see no font matching "(X?)Charter" on either Windows 10 or Arch Linux by default.  Do web browsers typically ship with additional fonts not found in the system?  I didn't see any changes trying to poke Charter into Firefox's web inspector (whereas I did see changes poking other fonts into it), so I assume Firefox doesn't provide it either; but then I'm not really that knowledgeable about web dev...

Reply:
I'll play around a bit more with this and maybe look at using something like Georgia that should be more widely available. I'm also really not picky about what font actually gets used as long as it's readable and looks half-decent 😛

Reply:
Same, it looks good on my system as it is.  I just wanted to point out what seemed to be a misplaced assumption in case there is a better solution.

Reply:
Updated to use Georgia, which actually seems a bit more readable (on macOS anyways).